ntp.go

```diff
@@ -333,19 +332,24 @@ func getTime(host string, opt QueryOptions) (*msg, error) {
 	m.setMode(client)
 	m.setVersion(opt.Version)
 	m.setLeapIndicator(LeapNotInSync)
-	xmt := toNtpTime(time.Now())
+
+	xmt_time := time.Now()
+	xmt := toNtpTime(xmt_time)
 	m.TransmitTime = xmt
 
 	err = binary.Write(con, binary.BigEndian, m)
 	if err != nil {
-		return nil, err
+		return nil, 0, err
 	}
 
 	err = binary.Read(con, binary.BigEndian, m)
 	if err != nil {
-		return nil, err
+		return nil, 0, err
 	}
 
+	dt41 := time.Since(xmt_time) // uses monotonic clock @ Go 1.9+
```

Comment:
Why is it named `dt41`?

 Also, idiomatic go doesn't use underscores.    `xmitTime` would be idiomatic.  

Can you please install golint and run it on your code?

Reply:
> Can you please install golint and run it on your code?

Thanks! I was unaware of it, I though that golint is a symbolic name for part of `go vet` + `go fmt` :)

Reply:
> Why is it named dt41?

Delta between t4 and t1.